.secrets.baseline

```diff
@@ -1,9 +1,9 @@
 {
   "exclude": {
-    "files": null,
+    "files": "package-lock.json",
     "lines": null
   },
```

Comment:
I updated this file to ignore the package-lock.json file.
See this: https://github.com/alphagov/gds-pre-commit/blob/master/usage.md
```
Exclude Files
If you have files which you do not wish to run the scan against, such as a package-lock.json or go.sum which are automatically generated and have a number of high entropy strings, you can use --exclude-files *your files* to ignore these during the scan, this will be noted in .secrets.baseline to prevent you having to add this argument during future runs.
```